src/ape/managers/converters.py

```diff
@@ -140,6 +141,27 @@ def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
 list_tuple_converter = ListTupleConverter(None, None, None)  # type: ignore
 
 
+class TimestampConverter(ConverterAPI):
+    """Converts seconds to date and time string value"""
+
+    def is_convertible(self, value: str) -> bool:
+        if not isinstance(value, str):
+            return False
+        if " " not in value or len(value.split(" ")) > 2:
+            return False
+        try:
+            datetime.strptime(value, "%m-%d-%Y %H:%M:%S")
```

Comment:
I wish we also supported `%Y-%m-%dT%H:%M:%S.%f`, as this is the format I see all over server timestamps

Reply:
I guess we would lose precision though... hmm...